### PR TITLE
React Router DOM: `routing.kt` components separation

### PR DESCRIPTION
**Pull request by aerialist7**
cc @turansky @sgrishchenko

### CODE REVIEW

File: kotlin-react-router-dom/src/main/kotlin/react/router/dom/History.kt [hunk 1]
```diff
@@ -0,0 +1,17 @@
+@file:JsModule("react-router-dom")
+@file:JsNonModule
+
+package react.router.dom
+
+// TODO: Move to separate "history" module
```
Comment (turansky) [inline kotlin-react-router-dom/src/main/kotlin/react/router/dom/History.kt:6]: Only this interface?

File: kotlin-react-router-dom/src/main/kotlin/react/router/dom/Route.kt [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+package react.router.dom
+
+import react.ComponentClass
+import react.ComponentType
+import react.Props
+import react.ReactElement
+
+external interface RouteProps : Props {
+    var path: Array<out String>
```
Comment (turansky) [inline kotlin-react-router-dom/src/main/kotlin/react/router/dom/Route.kt:12]: `ReadonlyArray`?

File: kotlin-react-router-dom/src/main/kotlin/react/router/dom/RouterProps.kt [hunk 3]
```diff
@@ -0,0 +1,10 @@
+@file:JsModule("react-router-dom")
```
Comment (turansky) [inline kotlin-react-router-dom/src/main/kotlin/react/router/dom/RouterProps.kt:1]: Is it required?

**Comment by Leonya** [comment]
Released in pre.240

**Pull request merged**